lib/class-factory.js

```diff
@@ -140,8 +140,9 @@ function ClassFactory (vm) {
     }
   });
 
-  this.use = function (className) {
-    let C = getUsedClass(className);
+  this.use = function (className, skipCache) {
```

Comment:
```suggestion
  this.use = function (className, options = {}) {
    const skipCache = options.cache === 'skip';
```

Reply:
Just to clarify, is your plan to have an enum option where the value of it would be `"skip"` so that this change continues to work? Or do you plan to change the interface later to introduce it? I'm fine with this sort of thing in general though. I just did the bool/undefined truthiness check mostly to avoid having too much overhead.

Reply:
My thinking is: `Java.use('foo', { cache: 'skip' })`

I also thought of the added overhead, but considering that `Java.use()` should only be called at startup (to set up some variables at the outermost scope), and very rarely from hooks (only when it's not known or available until then), the added overhead shouldn't matter for well-written scripts.

But ISTM that the better solution here is to tie the cache to the class loader. So if you change the class loader it should also swap out the cache. But then OTOH there's another challenge: class-loaders are chained, and we don't want to re-generate the wrappers for `java.lang.*` and Android system classes. So perhaps this solution is simpler? And it solves the case where we want to get a wrapper for a class without keeping it in memory forever.

Reply:
Yeah, as mentioned above, this was a quick hack to ignore the fact that the it's just the one global cache across arbitrary classloaders. My take on this is that the ideal solution would be to have multiple separate class factory instances that could be set up w/ their own classloader and then have their own cache instead of having the global class factory and swapping in and out the classloader, which is a bit race-y. I get your concerns about the potential overhead though, though I don't know enough about the inner workings of the logic to say much.